beeai/agents/triage_agent.py

```diff
@@ -404,16 +409,14 @@ async def determine_target_branch_step(state):
             return Workflow.END
 
         async def comment_in_jira(state):
+            comment_text=state.triage_result.model_dump_json(indent=4)
+            logger.info(f"Result to be put in Jira comment: {comment_text}")
```

Comment:
@siteshwar I did this change as the attributes of the class vary depending on the resolution and this could also help with debugging (having all info about the result), wdyt?